### ISSUE TITLE
Signup page: The chat button placeholder has margin-left, which seems odd 

### ISSUE THREAD

**Issue by Abwasserrohr** [OdyseeTeam/odysee-frontend#567]
It's easier to show in a video I think, but there should not be that much space, I think:

https://user-images.githubusercontent.com/44294954/147493000-4c7eb637-4cc4-45c6-83c5-2e791dea4d44.mp4

**Comment by tzarebczan** [comment]
Good catch, have seen this one as well but it hasn't annoyed me enough to file it because I kept telling myself I'd fix it later lol
